packages/babel-plugin-transform-bigint/test/fixtures/addition/expected.js

```diff
@@ -0,0 +1,2 @@
+import _BigInt from "bigint-polyfill";
```

Comment:
Could even have an option for diff library if you wanted, if it passes the tests we make it's all good